### ISSUE TITLE
Update versions of dependencies

### ISSUE THREAD

**Issue by jmanthony3** [JuliaIntervals/IntervalRootFinding.jl#201]
Would it be possible to update the version numbers of the `IntervalArithmetic` and `Polynomials` dependencies?

```julia-repl
(@v1.10) pkg> status --outdated
⌅ [d1acc4aa] IntervalArithmetic v0.20.9 (<v0.22.11): IntervalRootFinding
⌅ [f27b6e38] Polynomials v3.2.13 (<v4.0.8): IntervalRootFinding
```

**Comment by Kolaru** [comment]
It is not possible currently, because we are still working on updating to the breaking changes of IntervalArithmetic. I have to check for Polynomials.

**Comment by OlivierHnt** [comment]
I think this is fixed by #203, since the dependency is no longer there.
